.github/workflows/ci.yml

```
@@ -0,0 +1,377 @@
+    needs: validate
+    
+    strategy:
+      matrix:
+        rust-version: [stable, beta]
+        
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Setup Rust toolchain
+      uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)